librosa/core/notation.py

```diff
@@ -346,15 +346,15 @@ def mela_to_svara(
     svara_map = [
         "Sa",
         "Ri\u2081",
-        None,  # Ri2/Ga1
-        None,  # Ri3/Ga2
+        "",  # Ri2/Ga1
+        "",  # Ri3/Ga2
         "Ga\u2083",
         "Ma\u2081",
         "Ma\u2082",
         "Pa",
         "Dha\u2081",
-        None,  # Dha2/Ni1
-        None,  # Dha3/Ni2
+        "",  # Dha2/Ni1
+        "",  # Dha3/Ni2
```

Comment:
easier to use empty strings here than to make the data type mixed between None and str.